camunda-formio-tasklist-vue/src/components/sort/TaskListSort.vue

```diff
@@ -37,10 +38,10 @@
         href="#"
         title="Ascending"
       >
-        <i class="fa fa-chevron-up cftf-arrow" aria-hidden="true"></i>
+        <i class="fa fa-chevron-up cftf-arrow ml-2" aria-hidden="true"></i>
       </a>
       <a v-else @click="toggleSort(idx)" href="#" title="Descending">
-        <i class="fa fa-chevron-down cft-arrow" aria-hidden="true"></i>
+        <i class="fa fa-chevron-down cft-arrow ml-2" aria-hidden="true"></i>
       </a>
     </b-row>
     <TaskSortOptions
```

Comment:
![image](https://user-images.githubusercontent.com/70306694/130733637-e740141e-99c3-491b-9820-b98147d128b8.png)

With the latest changes the + icon is looking not alligned with other icons